### PR TITLE
New command: skuba addon refresh localconfig

### PR DESCRIPTION
**Pull request by jenting**
# Describe your changes

Add a new command for skuba: `skuba addon refresh localconfig` in order to make local cluster definition folder in-sync with current Kubernetes version.

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Fixes: https://github.com/SUSE/avant-garde/issues/1765

### CODE REVIEW

**Comment by jenting** [comment]
skuba PR merged.

File: adoc/admin-updates.adoc [hunk 1]
```diff
@@ -172,6 +175,12 @@ That warning means that Cilium is going to regenerate all internal data on the f
 It can result in temporary connection loss for pods and services which might take few minutes.
 ====
 
+Updates the local cluster definition cluster addons base manifests. Please run:
```
Comment (c3y1huang) [inline adoc/admin-updates.adoc:178]: This is more of a `how` but not `why`. I would suggest to add some explanation to clarify in what situation this needs to be run.

**Comment by jordimassaguerpla** [comment]
This is not in 5.0.0.

**Comment by r0ckarong** [comment]
4.5.2 done with 877850f
4.2.5 prepared with 176dd5a

**Pull request merged**